hawc/apps/lit/api.py

```diff
@@ -379,26 +379,28 @@ def get_queryset(self):
     @action(detail=True, methods=("post",))
     def tag(self, request, pk):
         response = {"status": "fail"}
-        ref = self.get_object()
-        assessment = ref.assessment
+        instance = self.get_object()
+        assessment = instance.assessment
         if assessment.user_can_edit_object(self.request.user):
             try:
                 tags = [int(tag) for tag in self.request.data.get("tags", [])]
             except ValueError:
                 return Response({"tags": "Array of tags must be valid primary keys"}, status=400)
-            resolved = ref.update_tags(request.user, tags)
+            resolved = instance.update_tags(request.user, tags)
             response["status"] = "success"
             response["resolved"] = resolved
         return Response(response)
 
     @action(detail=True, methods=("post",))
     def resolve_conflict(self, request, pk):
```

Comment:
I modified this; REST APIs should generally return JSON unless otherwise specified by the content type via content negotiation, generally we'd want to build a custom renderer that could return json or html depending on what's requested.